src/Features/CSharp/Portable/EditAndContinue/SyntaxComparer.cs

```diff
@@ -790,27 +790,47 @@ protected override bool TryComputeWeightedDistance(SyntaxNode leftNode, SyntaxNo
                     }
 
                 case SyntaxKind.UsingStatement:
-                    var leftUsing = (UsingStatementSyntax)leftNode;
-                    var rightUsing = (UsingStatementSyntax)rightNode;
-
-                    if (leftUsing.Declaration != null && rightUsing.Declaration != null)
                     {
```

Comment:
view with whitespace off.  This was just put in a block so that it and the next case section coudl both use `leftUsing/rightUsing`.